addons/html_editor/static/src/main/toolbar/toolbar_plugin.js

```diff
@@ -36,7 +36,7 @@ import { closestElement } from "@html_editor/utils/dom_traversal";
  * @property {string} groupId Id of a toolbar group
  * @property {string} commandId
  * @property {Object} [commandParams] Passed to the command's `run` function
- * @property {TranslatedString} [title] * - becomes the button's title (and tooltip content)
+ * @property {(selection: EditorSelection, nodes: Node[]) => TranslatedString} [title] * - becomes the button's title (and tooltip content)
```

Comment:
OR TranslatedString.
I think it would be like
```js
{TranslatedString | (selection: EditorSelection, nodes: Node[]) => TranslatedString} [title]
```